File: examples/language/bench/bench.py
import sys

nargs = len(sys.argv)
program_name = sys.argv[0]
if nargs < 2:
  print("usage: py "+program_name+" count")
  exit(0)
count_string = sys.argv[1]     
count = int(count_string)
for i in range(count):
  j = i * i

File: examples/language/bench_test/bench.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char** argv) {
  if (argc < 2) {
    printf("usage: %s count\n", argv[0]);
    exit(0);
  }
  int count = atoi(argv[1]);
  int j;
  for (int i = 0; i < count; i++)
    j = i * i;
}
